src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
   // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = this.hidden === false;
+  }
 }
 
 class Carnivore extends Animal {
   // write your code here
+  bite(name) {
```

Comment:
it's an object, but name implies it is sort of string

```suggestion
  bite(animal) {
```